ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -152,11 +152,10 @@ class GetBlobOperation extends GetOperation {
    * @param compressionService The compression service to use.
    */
   GetBlobOperation(RouterConfig routerConfig, NonBlockingRouterMetrics routerMetrics, ClusterMap clusterMap,
-      ResponseHandler responseHandler, BlobId blobId, GetBlobOptionsInternal options,
-      Callback<GetBlobResult> callback, RouterCallback routerCallback, BlobIdFactory blobIdFactory,
-      KeyManagementService kms, CryptoService cryptoService, CryptoJobHandler cryptoJobHandler, Time time,
-      boolean isEncrypted, QuotaChargeCallback quotaChargeCallback, AmbryCache blobMetadataCache,
-      NonBlockingRouter nonBlockingRouter, CompressionService compressionService) {
+      ResponseHandler responseHandler, BlobId blobId, GetBlobOptionsInternal options, Callback<GetBlobResult> callback,
```

Comment:
try to avoid these formatting changes, they expand the PR unnecessarily

Reply:
This format is changed by IntelliJ.  I apply reformatting to the entire file.